scripts/jobs/env_services/alloy_api_export.py

```diff
@@ -41,17 +50,26 @@ def create_s3_key(s3_downloads_prefix, import_date, file, prefix_to_remove=None)
     file_table_name = file_table_name[0]
     file_table_name = file_table_name.lower()
 
-    for pre in prefix_to_remove:
-        pre = pre.lower()
+    if prefix_to_remove == None:
+        pass
+    else:
+        pre = prefix_to_remove.lower()
+
         if not file_table_name.startswith(pre):
             pass
         else:
-            file_table_name = file_table_name[len(pre):]
+            file_table_name = file_table_name[len(pre) :]
```

Comment:
> PEP 8 recommends to treat : in slices as a binary operator with the lowest priority, and to leave an equal amount of space on either side, **except if a parameter is omitted (e.g. ham[1 + 1 :])**. It recommends no spaces around : operators for “simple expressions” (ham[lower:upper]), and **extra space for “complex expressions” (ham[lower : upper + offset])**. **Black treats anything more than variable names as “complex” (ham[lower : upper + 1]).** It also states that for extended slices, both : operators have to have the same amount of spacing, except if a parameter is omitted (ham[1 + 1 ::]). Black enforces these rules consistently.

> This behaviour may raise E203 whitespace before ':' warnings in style guide enforcement tools like Flake8. **Since E203 is not PEP 8 compliant, you should tell Flake8 to ignore these warnings**.